scanner/scanner.py

```diff
@@ -0,0 +1,870 @@
+                        help="Name of the object you want to confirm",
+                        type=str,
+                        choices=["all", "timedelta", "manifests", "images"],
+                        default=None,
+                        required=True)
+    parser.add_argument("--versions",
+                        help="Java Versions",
+                        nargs='+',
+                        type=str,
+                        choices=["8", "11", "14"],
+                        default=["8", "11", "14"])
+    parser.add_argument("--jvms",
+                        help="Name of the JVMs",
+                        nargs='+',
+                        type=str,
+                        choices=["hotspot", "openj9"],
+                        default=["hotspot", "openj9"])
+    parser.add_argument("--oss",
+                        help="Names of the OSs",
+                        nargs='+',
+                        type=str,
+                        choices=["alpine", "debian", "debianslim", "ubi", "ubi-minimal", "centos", "clefos", "ubuntu"],
+                        default=["alpine", "debian", "debianslim", "ubi", "ubi-minimal", "centos", "clefos", "ubuntu"])
+    parser.add_argument("--packages",
+                        help="Names of the Packages",
+                        nargs='+',
+                        type=str,
+                        choices=["jdk", "jre"],
+                        default=["jdk", "jre"])
+    parser.add_argument("--archs",
+                        help="Architectures",
+                        nargs='+',
+                        type=str,
+                        choices=["armv7l", "aarch64", "ppc64le", "s390x", "x86_64"],
+                        default=["armv7l", "aarch64", "ppc64le", "s390x", "x86_64"])
+    parser.add_argument("--builds",
+                        help="Name of the Builds",
+                        nargs='+',
+                        type=str,
+                        choices=["slim", "full"],
+                        default=["slim", "full"])
+    parser.add_argument("--filter-bad-manifests",
+                        help="Filter out bad manifest images",
+                        action="store_true",
+                        default=False)
+    parser.add_argument("--delta-hours",
+                        help="Number of hours to deem an image 'old'",
+                        type=int,
+                        default=2)
+    parser.add_argument("--force-old-images",
+                        help="Force old images not to be filtered out",
+                        action="store_true",
+                        default=False)
+    parser.add_argument("--debug",
+                        help="Enable Debug output",
+                        action="store_true",
+                        default=False)
+    parser.add_argument("--log-path",
+                        help="Path to where the log file will be generated",
+                        type=str,
+                        default=None)
```

Comment:
Can we also add an parameter for listing valid images as seen by this script. Would be nice to cross-check it with the other scripts in the repo.

Reply:
Ya, I think this is a good idea. Let me see if I can rework it to all for that output.